frontend/.gitignore

```diff
@@ -13,6 +13,8 @@ pids
 *.seed
 *.pid.lock
 
+.yarnrc
+
```

Comment:
We need to add here the .yarn folder: _.yarn/_
And remove the _.yarnrc_ and _.yarn_ folder from the commit.